extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -936,6 +938,13 @@ export class AmpLightboxGallery extends AMP.BaseElement {
     };
 
     const mutate = () => {
+      if (enter) {
+        Services.ownersForDoc(this.element)./*OK*/ scheduleUnlayout(
```

Comment:
Can you comment why you're calling `unlayout` when you're entering? Shouldn't this have been called when we closed the lightbox?

Reply:
Good call, this implementation doesn't appear to clean up after itself well.